docs/source/user_guide/data_checks.ipynb

```diff
@@ -167,6 +319,91 @@
     "        warning_msg = \"Column '{}' has zero variance\"\n",
     "        return [DataCheckError(warning_msg.format(column), self.name) for column in X.columns if len(X[column].unique()) == 1]"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data Checks\n",
+    "\n",
+    "For convenience, EvalML provides a `DataChecks` class to represent a collection of data checks. We will go over `DefaultDataChecks`, a collection defined and used in `AutoMLSearch.`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Default Data Checks\n",
+    "\n",
+    "By default, `AutoMLSearch.search` runs a collection of data checks before it searches and iterates over pipelines. This collection of data checks is stored in the `DefaultDataChecks` class. It consists of a few data checks that are generally helpful for any machine learning problem. They are:\n",
+    "\n",
+    "* `HighlyNullDataCheck`\n",
+    "* `IDColumnsDataCheck`\n",
+    "* `TargetLeakageDataCheck`\n",
+    "* `InvalidTargetDataCheck`\n",
+    "* `ClassImbalanceDataCheck`\n",
+    "* `NoVarianceDataCheck`"
```

Comment:
Looks good. We'll have to remember to update this list, and/or simply link to the API ref instead.

Reply:
Doing both, added an API ref link 😁